### PR TITLE
Document how to install everything manually on Debian

### PR DESCRIPTION
**Pull request by hmpf**
Closes https://github.com/Uninett/campus-tasks/issues/27

### CODE REVIEW

**Review by johannaengland — COMMENTED** [review]
I have not tried it out, only did some proofreading

File: docs/source/install-on-debian.rst [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+    .. code-block:: console
+
+        $ sudo useradd -m -s /bin/bash howitz
+        $
+
+   This creates a user "howitz" with the group "howitz" which owns the new
+   directory "/home/howitz". The user has (and needs) no password and the shell
+   is set to bash. The "howitz"-user should NOT have root/sudo-access.
+
+#. Become howitz and create a venv:
```
Comment (lunkwill42) [inline docs/source/install-on-debian.rst:51]: Depending on how familiar someone reading this is with Python, the term `venv` might not mean much to many people.  Maybe better to use the term "virtual environment" or "Python virtual environment", and use "venv" as an abbreviation of the term later.

File: docs/source/install-on-debian.rst [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+
+Install via pip
+---------------
+
+TBD
+
+Configure howitz
+================
+
+See howitz's README.rst for how to configure howitz and create a user.
```
Comment (lunkwill42) [inline docs/source/install-on-debian.rst:140]: Not for this PR, I guess, but the README file could stand to have a more in-depth explanation of the double user database paradigm:  I.e. you don't use your Zino user directly in Howitz, but all Howitz users must be mapped to a Zino user....

File: docs/source/install-on-debian.rst [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+
+User database
+-------------
+
+Change ``[howitz] storage`` to ``/home/howitz/howitz.sqlite3``. Then ensure
```
Comment (lunkwill42) [inline docs/source/install-on-debian.rst:145]:
```suggestion
Change ``storage`` in the ``[howitz]`` section to ``/home/howitz/howitz.sqlite3``. Then ensure
```

File: docs/source/install-on-debian.rst [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+We recommend storing the debug-log in ``/var/log/howitz/debug.log``.
+
+#. Create the directory ``/var/log/howitz``:
+
+    .. code-block:: console
+
+        $ sudo mkdir var/log/howitz
+        $
+
+#. Change ``[logging.handlers.debug] filename`` to
```
Comment (lunkwill42) [inline docs/source/install-on-debian.rst:178]:
```suggestion
#. Change ``filename`` in the ``[logging.handlers.debug]`` section to
```

File: docs/source/install-on-debian.rst [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+TADA!
+
+Troubleshooting
+===============
+
+I get a "500 Server Error" page instead of the howitz landing page!
+-------------------------------------------------------------------
+
+Did you remember to configure howitz? Double-check that you've set the
+``[flask] SECRET_KEY`` and ``[zino.connection.default] server``. The values are
```
Comment (lunkwill42) [inline docs/source/install-on-debian.rst:340]:
```suggestion
Did you remember to configure howitz? Double-check that you've set
``SECRET_KEY`` in the ``[flask]`` section and ``server`` in the ``[zino.connection.default]`` section. The values are
```

**Review by lunkwill42 — CHANGES_REQUESTED** [review]
Looks pretty comprehensive to me :)  I've read it, but I haven't test-driven it.

File: docs/source/install-on-debian.rst (hunk 2 above)
Comment (hmpf) [reply docs/source/install-on-debian.rst]: I've put that into this doc and can move it elsewhere later.

**Review by lunkwill42 — APPROVED** [review]
Good enough for me.  Will maybe have to do a run-through some other day, the weekend is looming :)

**Pull request merged**